python/dgl/heterograph.py

```diff
@@ -867,67 +851,124 @@ def is_unibipartite(self):
 
     @property
     def ntypes(self):
-        """Return the list of node types of this graph.
+        """Return the node types of the graph.
 
         Returns
         -------
         list of str
+            Each ``str`` is a node type.
 
         Examples
         --------
+        The following example uses PyTorch backend.
+
+        >>> import dgl
+        >>> import torch
 
-        >>> follows_g = dgl.graph(([0, 1], [1, 2]), 'user', 'follows')
-        >>> plays_g = dgl.bipartite(([0, 1, 1, 2], [0, 0, 1, 1]), 'user', 'plays', 'game')
-        >>> g = dgl.hetero_from_relations([follows_g, plays_g])
+        >>> g = dgl.heterograph({
+        >>>     ('user', 'follows', 'user'): (torch.tensor([0, 1]), torch.tensor([1, 2])),
+        >>>     ('user', 'follows', 'game'): (torch.tensor([0, 1, 2]), torch.tensor([1, 2, 3])),
+        >>>     ('user', 'plays', 'game'): (torch.tensor([1, 3]), torch.tensor([2, 3]))
+        >>> })
         >>> g.ntypes
-        ['user', 'game']
+        ['game', 'user']
         """
         return self._ntypes
 
     @property
     def etypes(self):
-        """Return the list of edge types of this graph.
+        """Return the edge types of the graph.
```

Comment:
Better to add a Note: if there are multiple etypes with same name, the order of them can not be reflected in this API , please use canonical_etypes.

Reply:
Updated.